sylvia-derive/src/input.rs

```diff
@@ -243,4 +220,28 @@ impl<'a> ImplInput<'a> {
     fn emit_glue_msg(&self, name: &Ident, msg_ty: MsgType) -> TokenStream {
         GlueMessage::new(name, self.item, msg_ty, &self.error).emit()
     }
+
+    fn emit_querier_for_bound_impl(
+        &self,
+        interfaces: &Interfaces,
+        variants: MsgVariants<'a>,
+    ) -> TokenStream {
+        let trait_module = interfaces
+            .interfaces()
+            .first()
+            .map(|interface| {
+                let module = &interface.module;
+                quote! { #module ::}
```

Comment:
I really don't like adding this `::` to the module - it is a minor thing, but on usage, i prefer `#module :: Item` far more than `#module Item`, as it is more obvious that module is some module path (not usage of it looks like it could be some `let` or another keyword). I know you are saving some common keystrokes, but I find it impacting readability too much.

Reply:
I also don't like it but here I had to do it because we need this module to be used also for QueryMsg later on.
We could extract the module into TokenStream and then merge with QueryMsg and Querier

Reply:
changed so that Option<&Path> is being passed to these methods and inside we merge the modules with proper types.